Makefile

```diff
@@ -49,9 +49,10 @@ unit: kubebuilder
 # Ensure kubebuilder is installed before continuing
 KUBEBUILDER_ASSETS_ERR := not detected in $(KUBEBUILDER_ASSETS), to override the assets path set the KUBEBUILDER_ASSETS environment variable, for install instructions see https://book.kubebuilder.io/quick-start.html
 kubebuilder:
-ifeq (, $(wildcard $(KUBEBUILDER_ASSETS)/kubebuilder))
-	$(error kubebuilder $(KUBEBUILDER_ASSETS_ERR))
-endif
+# not needed for envtest
+# ifeq (, $(wildcard $(KUBEBUILDER_ASSETS)/kubebuilder))
+# 	$(error kubebuilder $(KUBEBUILDER_ASSETS_ERR))
+# endif
```

Comment:
Do we need to still provide a Makefile target that installs envtest? That target could then be a pre-requisite for the `unit` target. Thoughts?

Reply:
yes I would vote for it. This would be for a separate PR.

Reply:
Yeah, that's sounds reasonable to me as long as we commit to tackling that in a follow-up PR.